app/javascript/components/App.js

```diff
@@ -2,14 +2,19 @@ import React from 'react';
 import { Route, Switch } from 'react-router-dom';
 import { withRouter } from 'react-router';
 import Home from "./home";
+import Parties from "./parties";
 import Program from "./program";
 
 class App extends React.Component {
+    constructor() {
+        super();
+    }
+
     render() {
         return (
             <Switch>
                 <Route exact path="/" component={Home} />
-                <Route exact path="/program/:party_acronym/:section_id?" component={withRouter(Program)} />
+                <Route exact path="/programa/:party_acronym/:section_id?" component={withRouter(Program)} />
```

Comment:
Lets move this to `manifesto`, ok?

Reply:
Will do. Just to mention, all paths should be in portuguese, right?

Reply:
I would say English